### ISSUE TITLE
Extension Platfrom?

### ISSUE THREAD

**Issue by Dangry98** [vilemduha/optiloops#11]
Hi, this is really useful addon, any plans to submit it to the New Extension Platform?
https://extensions.blender.org/

**Comment by vilemduha** [comment]
Yes, it will be as an extension. Currently porting this to our shared repos here:
https://github.com/BlenderKit/Optiloops

**Comment by xdanielc** [comment]
Any news on this? I still don't see it on the extensions platform
